Screen mock

belmeetmule opened the pull request:
Hello Bro,

In this PR, I just added the app's screen mock png and added it to the readme.md.

belmeetmule committed:
add screen mock image

belmeetmule committed:
Update readme to show screen mock image

Wahaj-Ali reviewed (approved):
Awesome work @belmeetmule 👏

Wahaj-Ali committed:
Update SideBar.css

Add media query to modify footer styles for mobile view.

Wahaj-Ali committed:
Update Houses.scss

Modify style for the house image on the main-page for a better-looking UI.

Wahaj-Ali committed:
Update addHouseSlice.js

Remove syntax error preventing from successfully deleting a house.

Wahaj-Ali committed:
Update deleteHouse.js

The app shows a message informing the user whenever a house is deleted from the list.

Wahaj-Ali committed:
Update Reservation.js

Modify the text for reservations page.

Wahaj-Ali committed:
Update HouseDetails.js

Add authorization, only logged in users can access the reservations page(only signed in users can book their visit).

Wahaj-Ali committed:
Update HouseDetails.scss

Style the disabled button when the user in not signed in.

Wahaj-Ali commented:
Hi @belmeetmule,
I have made some changes including
- a modified style for the sidebar in the mobile view.
- Update the book visit functionality, now available only to the logged-in users.

Please leave your remarks.

Wahaj-Ali committed:
Update HouseDetails.scss

Decrease font-sizes for the mobile view.
